scripts/e2e-samples.sh

```diff
@@ -34,9 +34,10 @@ if [[ "${BACON_TASK}" == true ]]; then
   get_vault_secret_key devex/trex-js-idx-sdk-vars trex_idx_sdk_e2e_apiKey OKTA_API_KEY
 else
   echo "Running tests against production (ok12) org"
+  export SPA_CLIENT_ID=0oa17suj5x9khaVH75d7
```

Comment:
what's this client id for?

Reply:
It's [`Javascript IDX SPA`](https://javascript-idx-sdk-admin.okta.com/admin/app/oidc_client/instance/0oa17suj5x9khaVH75d7/#tab-general) app to be used in specs tests instead of web app